### PR TITLE
Add Azure details to the splunk returners

### PR DESCRIPTION
**Pull request by mew1033**

### CODE REVIEW

**Comment by mew1033** [comment]
So far all we're pulling is the vmId.

**Comment by basepi** [comment]
I'm wondering if we should combine the details modules into a single `cloud_details.py` module. Thoughts?

**Comment by mew1033** [comment]
Not a bad idea. I'm fairly certain we'll be adding in Ethos in the somewhat near future. Might be good to have them all in one.

**Comment by basepi** [comment]
Would you mind tackling putting them into a single module, @mew1033? That is, if you have time. Would be nice to do it now rather than later.

**Comment by mew1033** [comment]
Okay, I think I got them combined. How does it look?

File: _returners/cloud_details.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    aws = get_aws_details()
+    azure = get_azure_details()
+
+                        if aws['aws_account_id'] is not None:
+                            event.update({'aws_ami_id': aws['aws_ami_id']})
+                            event.update({'aws_instance_id': aws['aws_instance_id']})
+                            event.update({'aws_account_id': aws['aws_account_id']})
+
+                        if azure['azure_vmId'] is not None:
+                            event.update({'azure_vmId': azure['azure_vmId']})"""
```
Comment (basepi) [inline _returners/cloud_details.py]: Can you remove this doc block? It appears to be out of date.

**Review by basepi — CHANGES_REQUESTED** [review]
Beautifully done. You left some old docs but otherwise I think this looks great!

**Comment by mew1033** [comment]
Oops, my bad. I had pasted that in there as a reference. Got it removed.

**Pull request merged**